spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -643,6 +679,142 @@ def main(argv: Sequence[str]):
 
+        # Register spinal rootlets to template:
+        if label_type == 'rootlet':
+
+            # Apply transformation to rootlets and image
+            sct_apply_transfo.main(argv=[
+                '-i', ftmp_rootlet,
+                '-o', add_suffix(ftmp_rootlet, '_straightAffine'),
+                '-d', ftmp_template,
+                '-w', 'warp_curve2straightAffine.nii.gz',
+                '-x', 'nn',
+                '-v', '0',
+            ])
+            ftmp_rootlet = add_suffix(ftmp_rootlet, '_straightAffine')
+
+            # Dilate rootlets masks:
+            src_mask = Image(dilate(Image(ftmp_rootlet), size=3, shape='ball'), hdr=Image(ftmp_rootlet).hdr).save(add_suffix(ftmp_rootlet, '_dil'))
+            src_mask = add_suffix(ftmp_rootlet, '_dil')
+            dest_mask = Image(dilate(Image(ftmp_template_rootlets), size=3, shape='ball'), hdr=Image(ftmp_template_rootlets).hdr).save(add_suffix(ftmp_template_rootlets, '_dil'))
+            dest_mask = add_suffix(ftmp_template_rootlets, '_dil')
+            src_im = ftmp_data
+            dest_im = ftmp_template
+            metricSize = '4'
+
+            cmd_rootlet = ['isct_antsRegistration',
+                           '--dimensionality', '3',
+                           '--transform', 'bsplinesyn' + '[' + step_rootlet.gradStep
+                           + ',26,0,3' + ']',
+                           '--metric', step_rootlet.metric + '[' + dest_im + ',' + src_im + ',1,' + metricSize + ']',
+                           '--convergence', step_rootlet.iter,
+                           '--shrink-factors', step_rootlet.shrink,
+                           '--smoothing-sigmas', step_rootlet.smooth + 'mm',
+                           '--restrict-deformation', step_rootlet.deformation,
+                           '--output', '[step' + str(step_rootlet.step) + ',' + add_suffix(ftmp_data, '_Rootlets_alldir') + ']',
+                           '--interpolation', 'linear',
+                           '--masks', '[' + dest_mask + ',' + src_mask + ']',
+                           '--verbose', ('1' if verbose >= 1 else '0'),
+                           ]
+            printv('\nRegistering with rootlets masks in z...', verbose)
+            printv(cmd_rootlet, verbose)
+            status, output = run_proc(cmd_rootlet, verbose, is_sct_binary=True)
+            printv(output, verbose)
+            if status != 0:
+                raise RuntimeError(f"Subprocess call {cmd_rootlet} returned non-zero: {output}")
+            printv('\nApply transformation after rootlets adjustment...', verbose)
+            # Average perslice warping field
+            cmd_split = ['sct_image', '-i', 'step10Warp.nii.gz', '-mcs']
+            status, output = run_proc(cmd_split, verbose, is_sct_binary=True)
+            printv(output, verbose)
+            # Compute slicewise mean in Z to ensure symmetry
+            img = Image('step10Warp_Z.nii.gz')
```

Comment:
Maybe we could an extra print to inform user that we're computing slicewise mean in Z. It took me some time to understand why we do `sct_image -mcs` and `sct_image -omc` as I did not imidietily noticed the difference `step10Warp_Z.nii.gz` vs `step10Warp_Z_mean.nii.gz`

<img width="2069" alt="image" src="https://github.com/user-attachments/assets/1873bb04-9702-45d2-b7f3-d962e589fd73" />


```suggestion
            # Compute slicewise mean in Z to ensure symmetry
            printv('\nComputing slicewise mean in Z to ensure symmetry....', verbose)
            img = Image('step10Warp_Z.nii.gz')
```

Reply:
Good idea!